staff/web/vhost/vhosting_check.py

```diff
@@ -43,14 +42,48 @@
     # OCF-Flag.png
     'ce48822dd63785f77d5826b6b215b18d',
     # lighter152x41.gif.png
-    'a9bfe3f918552692f5eddbd3c5446367'}
+    'a9bfe3f918552692f5eddbd3c5446367',
+    # Below hashes are calculated as of 2/18/18
+    # berknow150x40.png
+    '61cb03fc17cf8a0651627ee78c4801a9',
+    # binnov-157x46.png
+    '41c167531b66bf22bfcce5f4271b337f',
+    # lighter152x41.png
+    'a9bfe3f918552692f5eddbd3c5446367',
+    # lighter177x48.png
+    '774c97d92960c1ac9070c28e0530316b',
+    # lighter202x54.png
+    'dd65572250059a2bc61f79baa4ee00d6',
+    # metal152x41.png
+    'abe6c541989915d3e97a2fadd8b84aaf',
+    # metal177x48.png
+    'a8eff8196bd0e3e3062e2f2526fd92ce',
+    # metal202x54.png
+    'd56997fc89c6b2fe796a26c291e6de25',
+    # ocf-hosted-penguin-dark.png
+    '5692198b270c0f86be5b06fe6ac298bd',
+    # ocf-hosted-penguin-dark.svg
+    '93c79a55e31311628f77e51058c2c0cb',
+    # ocf-hosted-penguin.png
+    '96304f6f203fa4538f10caeffaea09b5',
+    # ocf-hosted-penguin.svg
+    '2aaf138a323e9f5a8ee98bafbb5615bf',
+    # ocf-hosting-flag-wave-250x122.png
+    'ce48822dd63785f77d5826b6b215b18d',
+    # ocfbadge_blue8.png
+    '0276436d44d1a5e7cfc94b0f2d96907f',
+    # ocfbadge_silver8-b.png
+    'fc743ade8109379db4ddb2d4cf2723f3',
+    # ocfbadge_silver8.png
+    'a69a0929a9621638dd5b94161954fa5a'
+}
 
 disclaimer_re = 'We\s*are\s*a\s*student\s*group\s*acting\s*independently\s*'\
-                'of\s*the\s*University\s*of\s*California.\s*We\s*take\s*full'\
+                'of\s*the\s*University\s*of\s*California.\s*(<[\s\S]*>)?\s*We\s*take\s*full'\
```

Comment:
Isn't `\s\S` equivalent to `.`? Or am I missing something?

And also, what's the motivation for this tag? A `<br>`?